content/800-guides/240-partner-api.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: 'Partner Database Provisioning & User Claim Flow'
+metaTitle: 'Partner Database Provisioning & User Claim Flow'
+description: 'Learn how to use the Prisma Postgres Partner API to provision and claim databases'
+sidebar_label: 'Partner API'
```

Comment:
Let's make sure we call this "management API" throughout the guide, as there's no separate Partner API.